nodejs-style-loader/style-loader-0.9.0-registry.npmjs.org.tgz

```diff
@@ -0,0 +1 @@
+../.git/annex/objects/Vw/P4/SHA256E-s24380--4dfa6ccd21a52f006bcf128407ce674598f459a266c72a199330df59ace7f2ff.org.tgz/SHA256E-s24380--4dfa6ccd21a52f006bcf128407ce674598f459a266c72a199330df59ace7f2ff.org.tgz
```

Comment:
This file should be added directly, but all of the _others_ should be using git annex - they're the wrong way around.